.travis.yml

```diff
@@ -15,11 +15,24 @@ before_install:
 
 script:
     - find \( -name \*.c -o -name \*.h \) -exec sh -c '! grep --color "^[[:space:]]*//" $@' find {} \+ && echo No line comments found...
-    - docker run -v $TRAVIS_BUILD_DIR:/source ${OS_TYPE} bash -c "set -e;
-        ${INSTALL_REQUIREMENTS};
+    - echo > build.sh "set -e;
         cd /source;
         export CCFLAGS='-fno-strict-aliasing -Wno-unknown-pragmas -Wno-missing-braces -Wno-unused-result -Wno-return-type -Wno-int-to-pointer-cast -Wno-parentheses -Wno-unused -Wno-unused-but-set-variable -Wno-cpp -P';
         export CC=gcc;
         ./bin/package;
-        ./bin/package make -S 2>&1 | tee out.log ;
-        grep -q 'error:' out.log && exit 1 || exit 0;"
+        ./bin/package make ast-ksh 2>&1 | tee out.log;
+        grep -q 'error:' out.log && exit 1;
+        ln -s /usr/bin/cat arch/$(sh bin/package)/bin/;
+        ln -s /usr/bin/tee arch/$(sh bin/package)/bin/;
+        ln -s /usr/bin/chmod arch/$(sh bin/package)/bin/;
```

Comment:
I also noticed that these commands aren't built by `bin/package make ast-ksh` when someone else suggested I do that when I pointed out it's silly to have to build the entire AST project just to build and test ksh. Apparently that person has never tried to run the unit tests. Not to mention there is obviously missing dependencies in the definition of the `ast-ksh` target. These symlinks are fine for now but obviously we'll want to remove this hack once we've changed the build tool chain.

Reply:
I agree that we should remove these symlinks later.